src/components/Filter/Buttons/index.jsx

```diff
@@ -8,7 +8,7 @@ export default function Buttons({ filters = [], handleCategoryFilter }) {
   const CategoryFilter = ({ filter, image, alt }) => (
     <button className='ml-4' onClick={() => handleCategoryFilter(filter)}>
       <Image
-        className='rounded-lg hover:opacity-70'
+        className='rounded-full shadow-2xl transition hover:scale-105 hover:shadow-black'
```

Comment:
Instead of repeating these classes identically in more than one place, why don't you define a variable in a CSS module file, and you may name it st like transition and then just call it with all your desire to classes. That would be cleaner and easier to use and read 👍

Reply:
ok i'll

Reply:
Done !!